doc/ad_code_values/biden/12/DN.md

```diff
@@ -0,0 +1,10 @@
+---
+value: DN
```

Comment:
DN10 is similar, with a variant of a minimum $10 suggestion

Reply:
It feels like we could label `Unknown12` as `Call to action` if we could put some more values against the rest.